app/src/main/java/org/hackillinois/android/api/Secret.java

```diff
@@ -0,0 +1,5 @@
+package org.hackillinois.android.api;
+
+public class Secret {
```

Comment:
Maybe we should consider adding this to the .gitignore so that there's no need to special case this file in every commit.